dependencies.gradle

```diff
@@ -2,10 +2,10 @@
 
 dependencies {
     implementation('com.github.GTNewHorizons:NotEnoughItems:2.7.29-GTNH:dev')
+    implementation("com.github.GTNewHorizons:GTNHLib:0.6.11:dev")
```

Comment:
This needs to be added to the `@Mod` declaration as a required dependency with the `@[0.6.11,)` version range